test/sharness/t0020-service-startup-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# MIT LICENSED
+#
+test_description="Test service startup and init functionality"
+
+. lib/test-lib.sh
+
+test_expect_success "ipfs is installed on this machine" '
+   ipfs help >ipfs_help.txt &&
+   egrep -i "^Usage" ipfs_help.txt >/dev/null  
+'
+
+test_expect_success "launch ipfs daemon" '
+   mkdir -p ../.test_ipfs &&
+   IPFS_PATH="../.test_ipfs" eval '"'"'ipfs init'"'"' &&
```

Comment:
use a proper tmp path with `mktemp -d` (`mktemp -d ipfs-XXXXXX`). Store name in a variable. Make sure you have a final_cleanup to remove it. You can use `-p` to make the temp directory relative to this folder if you prefer, but don't store stuff in parent directories which don't belong to sharness tests.